schedule/lvm_thin_provisioning.yml

```diff
@@ -0,0 +1,77 @@
+name:           lvm_thin_provisioning
+description:    >
+    Complete OS deployment with unencrypted LVM drive management.
+    Test creates 2 LVM and BIOS boot partitions.
+    Thin pool and thin lv resides on the second LVM partition,
+    where /home (XFS) is being mounted. Maintainer: mloviska, poo#39023
+conditions:
+    bootloader:
+        ARCH:
+            x86_64:
+                - installation/isosize
+                - installation/bootloader
+            s390x:
+                - installation/bootloader_zkvm
+            aarch64:
+                - installation/isosize
+                - installation/bootloader
+    addons_repos:
+        DISTRI:
+            opensuse:
+                - installation/online_repos
+                - installation/installation_mode
+                - installation/logpackages
+            sle:
+                - installation/scc_registration
+                - installation/addon_products_sle
+    hostname_inst:
+        DISTRI:
+            sle:
+                - installation/hostname_inst
+    user_settings_root:
+        DISTRI:
+            sle:
+                - installation/user_settings_root
+    reconnect_mgmt_console:
+        ARCH:
+            s390x:
+                - boot/reconnect_mgmt_console
+    sle15_workarounds:
+        DISTRI:
+            sle:
+                - console/sle15_workarounds
+    opensuse_repos:
+        DISTRI:
+            opensuse:
+                - update/zypper_clear_repos
+                - update/zypper_ar
+                - update/zypper_ref
+
+modules:
```

Comment:
Another option, call this `scheduling` and the other one `conditional_scheduling` but it is longer too :)

Reply:
I've renamed it to `schedule` and `conditional_schedule`. We can leave it for now, at least with the feedback we got, it's not clear from the first sight. And it's good to get this feedback, as even we will document it, it should be clear without it.